### PR TITLE
protoc: add page

### PR DESCRIPTION
**Pull request by MatanRubin**
----

- [X] The page (if new), does not already exist in the repo.

- [X] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [X] The page has 8 or fewer examples.

- [X] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [X] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/protoc.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# protoc
+
+> Parse Google Protobuf `.proto` files and generate output in the specified language.
+
+- Generate Python code from a `.proto` file:
+
+`protoc --python_out={{output_directory}} {{input_file}}`
```
Comment (agnivade) [inline pages/common/protoc.md]: Change `output_directory` to `path/to/output_directory` everywhere.

Also change `input_file` to `input_file.proto` for clarity.

File: pages/common/protoc.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Generate Java code from a `.proto` file that imports other `.proto` files:
+
+`protoc --java_out={{output_directory}} --proto_path={{import_search_path}} {{input_file}}`
+
+- Generate Java code from a `.proto` file that imports other `.proto` files:
```
Comment (agnivade) [inline pages/common/protoc.md]: Repeated. We can remove this.

File: pages/common/protoc.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+`protoc --java_out={{output_directory}} --proto_path={{import_search_path}} {{input_file}}`
+
+- Generate code for multiple languages:
+
+`protoc --csharp_out={{c#_output_diry}} --js_out={{javascript_output_dir}} {{input_file}}`
```
Comment (agnivade) [inline pages/common/protoc.md]: Let's use a consistent naming convention. Change to `path/to/c#_output_directory` and `path/to/js_output_directory`.

File: pages/common/protoc.md (hunk 1 above)
Comment (MatanRubin) [reply pages/common/protoc.md]: fixed

File: pages/common/protoc.md (hunk 2 above)
Comment (MatanRubin) [reply pages/common/protoc.md]: Whoops, no idea how that happened :) 
fixed

**Comment by sbrl** [comment]
Thanks, @MatanRubin!

**Pull request merged**